__tests__/unit/auth/auth.test.js

```diff
@@ -1,46 +1,7 @@
 import jsonwebtoken from "jsonwebtoken";
```

Comment:
**This commit effectively removes any tests of auth.js.**
And from two cases left only one actually tests anything.

I created a card (no 67) on Trello to restore the tests and assigned you to it.